zerver/lib/response.py

```diff
@@ -105,10 +105,6 @@ def json_success(request: HttpRequest, data: Mapping[str, Any] = {}) -> MutableJ
     return json_response(data=data)
 
 
-def json_partial_success(request: HttpRequest, data: Mapping[str, Any] = {}) -> MutableJsonResponse:
-    return json_response(res_type="partially_completed", data=data, status=200)
```

Comment:
I thought it better to just completely remove this as a response option, and to have the endpoints return `json_success` with the relevant data if it times out before processing all the targeted messages.

Reply:
Yeah, that seems correct given the plan here.